src/lambdas/initiateAthenaQuery/handler.spec.ts

```diff
@@ -55,4 +67,23 @@ describe('initiate athena query handler', () => {
     )
     expect(mockConfirmAthenaTable).not.toHaveBeenCalled()
   })
+
+  it('updates zendesk and throws an error if no query sql is generated', async () => {
+    mockConfirmAthenaTable.mockResolvedValue({
+      tableAvailable: true,
+      message: 'test message'
+    })
+    mockCreateQuerySql.mockReturnValue({
+      sqlGenerated: false,
+      error: 'sql error message'
+    })
+
+    await expect(handler(testAthenaQueryEvent)).rejects.toThrow(
+      'sql error message'
+    )
+    expect(mockGetQueryByZendeskId).toHaveBeenCalled()
+    expect(mockConfirmAthenaTable).toHaveBeenCalled()
```

Comment:
Can you use `toHaveBeenCalledWith` here, and assert on the parameters you're expecting?

Reply:
Done